plugins/woocommerce/src/Internal/Admin/WcPaySubscriptionsPage.php

```diff
@@ -0,0 +1,266 @@
+			'parent'     => 'woocommerce',
+			'path'       => '/subscriptions',
+			'capability' => 'manage_options',
+		);
+
+		wc_admin_register_page( $menu_data );
+
+		if ( ! isset( $submenu['woocommerce'] ) || 'yes' === get_option( $this->user_viewed_option, 'no' ) ) {
+			return;
+		}
+
+		// translators: Admin menu item badge. It is used alongside the "Subscriptions" menu item to grab attention and let merchants know that this is a new offering.
+		$new_badge_text = __( 'new', 'woocommerce' );
+
+		// Add the "new" badge.
+		foreach ( $submenu['woocommerce'] as $key => $menu_item ) {
+			if ( 'wc-admin&path=/subscriptions' === $menu_item[2] ) {
+				$submenu['woocommerce'][ $key ][0] .= sprintf( ' <span class="wcpay-subscriptions-menu-badge awaiting-mod count-1"><span class="plugin-count">%s</span></span>', esc_html( $new_badge_text ) ); // phpcs:ignore WordPress.WP.GlobalVariablesOverride.Prohibited
+				break;
+			}
+		}
+	}
+
+	/**
+	 * Returns true if the store is eligible for the WooCommerce subscriptions empty state experiment.
+	 *
+	 * @return bool
+	 */
+	private function is_store_experiment_eligible() {
+		// Ineligible if WooCommerce Payments OR an existing subscriptions plugin is installed.
+		$installed_plugins      = PluginsHelper::get_installed_plugin_slugs();
+		$plugin_ineligible_list = array(
+			'woocommerce-payments',
+			'woocommerce-subscriptions',
+			'subscriptio',
+			'subscriptions-for-woocommerce',
+			'subscriptions-for-woocommerce-pro',
+			'sumosubscriptions',
+			'yith-woocommerce-subscription',
+			'xa-woocommerce-subscriptions',
+		);
+		foreach ( $plugin_ineligible_list as $plugin_slug ) {
+			if ( in_array( $plugin_slug, $installed_plugins, true ) ) {
+				return false;
+			}
+		}
+
+		// Ineligible if store address is not compatible with WCPay Subscriptions (US).
+		$store_base_location = wc_get_base_location();
+		if ( empty( $store_base_location['country'] ) || 'US' !== $store_base_location['country'] ) {
+			return false;
+		}
+
+		// Ineligible if store has not been active for at least 6 months.
+		if ( ! WCAdminHelper::is_wc_admin_active_in_date_range( 'month-6+' ) ) {
+			return false;
+		}
+
+		// Ineligible if store has not had any sales in the last 30 days.
+		if ( ! $this->get_store_recent_sales_eligibility() ) {
+			return false;
+		}
+
+		return true;
+	}
+
+	/**
+	 * Returns true if the store has an order that has been paid within the last 30 days.
+	 *
+	 * @return bool
+	 */
+	private function get_store_recent_sales_eligibility() {
+		$transient_key = 'woocommerce-wcpay-subscriptions_recent_sales_eligibility';
+
+		// Load from cache.
+		$is_eligible_cached = get_transient( $transient_key );
+
+		// Valid cache found.
+		if ( false !== $is_eligible_cached ) {
+			return wc_string_to_bool( $is_eligible_cached );
+		}
+
+		// Get a single order that has been paid within the last 30 days.
+		$orders = wc_get_orders(
+			array(
+				'date_paid' => '>' . strtotime( '-30 days' ),
```

Comment:
let's use `date_created` instead of paid since `date_paid` is stored in the metadata, so this query will be slower. you can add another filter on status for `processed` and `completed` if you'd like to get only orders that are paid.

Reply:
👍 can do.

I've changed to use the `'date_created'` and `'status'` args instead (https://github.com/woocommerce/woocommerce/pull/32957/commits/3aa2294968cfdf8d1896681210fa2daa0cfb8e16)